src/components/Files3d.vue

```diff
@@ -0,0 +1,111 @@
+<!--
+ - @copyright Copyright (c) 2019 Vinzenz Rosenkranz <vinzenz.rosenkranz@posteo.de>
+ -
+ - @author Vinzenz Rosenkranz <vinzenz.rosenkranz@posteo.de>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<template>
+	<div id="threejs-container">
+
+	</div>
+</template>
+
+<script>
+import Vue from 'vue'
+
+import {
+	PCFSoftShadowMap,
+	PerspectiveCamera,
+	Scene,
+	Vector3,
+	WebGLRenderer
+} from 'three';
+
+export default {
+	name: 'Files3d',
+	props: ['mime', 'path'],
```

Comment:
All of those are the available props:
https://github.com/nextcloud/viewer/blob/b16ba3ab0dc81a9e98a88a9917936c527e94cca6/src/mixins/Mime.js#L29-L86

Reply:
Alright, thanks. Is the readme outdated? There are only path and mime mentioned

Reply:
Well, you theoritecally do'nt really need more than a few, but yes, I guess it needs an update, sorry abou that :)